src/app/static/src/app/components/Warning.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+    <div v-if="warnings.length > 0">
+        <h4 class="alert-heading pt-2">
+            <alert-triangle-icon size="1.5x" class="custom-class mr-1 mb-1"></alert-triangle-icon>
+            <span v-translate="headerText(origin)"></span>
+        </h4>
```

Comment:
One thing that wasn't clear after speaking with Jeff was whether the headers should be smaller. Easy to change if they need to be though.